src/pybind/mgr/dashboard/ci/cephadm/ceph_cluster.yml

```diff
@@ -4,14 +4,14 @@ parameters:
  network: ceph-dashboard
  domain: cephlab.com
  prefix: ceph
- numcpus: 1
+ numcpus: 8
  memory: 2048
  image: fedora34
  notify: false
  admin_password: password
  disks:
- - 15
- - 5
+ - 25
+ - 25
```

Comment:
not needed.

Reply:
> not needed.

Ah i pushed these changes by mistake..will remove. Thanks @nizamial09